.github/workflows/build_test_deploy.yml

```
@@ -1,13 +1,431 @@
+      #
+      - uses: docker/setup-buildx-action@v2
+      - uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKER_USER }}
+          password: ${{ secrets.DOCKER_PASS }}
+      - name: Build webknossos docker image
+        uses: docker/build-push-action@v6
+        with:
+          context: .
+          push: true
+          tags: |
```

actionlint (1.7.4)

property "build" is not defined in object type {}

(expression)